pkg/utils/rand/rand.go

```diff
@@ -0,0 +1,29 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rand
+
+import (
+	"encoding/base32"
+	"math/rand"
+)
+
+func String(length int) string {
+	label := make([]byte, 32)
+	_, err := rand.Read(label) //nolint
```

Comment:
Crashes for length >= 57, `make([]byte,length)` will generate more than enough data to get `length` characters out of it. Not a big deal at the moment since it's not called with large lengths.